src/components/file-input/UIFileInput.vue

```diff
@@ -138,6 +140,14 @@ export default defineComponent({
          */
         maxFiles: positiveOptionalNumber,
 
+        /**
+         * Override files preview output.
+         */
+        displayNameFunc: {
+            type: Function,
```

Comment:
you could type this as follows: `as PropType<(file: MaybeArray<File> | null) => string>`
this will give us typehints inside and down the line outside of the component